scrapers/zfilmzoriginals.yml

```diff
@@ -0,0 +1,27 @@
+name: z-filmz-originals
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - z-filmz-originals.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="info-section--title"]/text()
+      Date:
+        selector: //span[@title="Published at"]
+        postProcess:
+          - parseDate: 2006-01-02
+      Performers:
+        Name: //span[@title="Models"]//a
+      Tags:
+        Name: //span[@title="Categories"]//a
```

Comment:
some users might want `all` the tags ( the categories ones are the ones i would also add be default)
maybe add the combined selector commented out?
```
//span[@title="Categories"]//a|//div[@class="tags mt-3"]/a
```